File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.py
def solution(age):
    return ''.join(chr(int(i)+97) for i in str(age))

# chr() : 아스키코드를 문자열로
# ord() : 문자열를 아스키코드로
# age 한글자씩 나눠야하니까 문자열로 바꿔서 for돌리고 각각 숫자로 바꾼후에 거기에 해당하는 문자열 출력해서 붙이기
# 아스키코드 A = 65 , a = 97

File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";
        
        String[] ageArr = String.valueOf(age).split("");
        
        for(int i=0; i<ageArr.length; i++){
            answer += (char)(Integer.parseInt(ageArr[i]) + 97); // 아스키코드 97 = a 
        }
        return answer;
    }
}